src/applications/SimulationSpecification.java

```diff
@@ -146,7 +146,7 @@ private void setupJob(Machine machine, MachineShopSimulator shop, int p) {
 	}
 
 	public  SimulationResults runSimulation(MachineShopSimulator machineShopSimulator) {
-	    machineShopSimulator.largeTime = Integer.MAX_VALUE;
+	    machineShopSimulator.setLargeTime(Integer.MAX_VALUE);
```

Comment:
Does largeTime need to be set here or can it just be initialized at the beginning? I think largeTime is only ever Integer.MAX_VALUE, so I think a setter that takes in a parameter is definitely bad.

Reply:
Yes I agree